server/model/json.go

```diff
@@ -126,9 +134,15 @@ func (r Run) MarshalJSON() ([]byte, error) {
 		CompletedAt:               r.CompletedAt,
 		TestVersion:               r.TestVersion,
 		Trigger:                   r.Trigger,
-		TriggerResult:             r.TriggerResult,
 		Trace:                     r.Trace,
 		Results:                   r.Results,
+		TriggerResult: encodedTriggerResult{
+			Type:    r.TriggerResult.Type,
+			HTTP:    r.TriggerResult.HTTP,
+			GRPC:    r.TriggerResult.GRPC,
+			SpanID:  r.TriggerResult.SpanID.String(),
```

Comment:
Why do we need to move the spanID/TraceID from the Run to the trigger?

Reply:
We either do this or we pass the pointer to the run so we can set the spanID and traceID to the run instance.